eng/testing/RunnerTemplate.sh

```diff
@@ -228,7 +230,7 @@ fi
 if [ -n "$HELIX_WORKITEM_PAYLOAD" ]; then
   # For abrupt failures, in Helix, dump some of the kernel log, in case there is a hint
   if [[ $test_exitcode -ne 1 ]]; then
-    dmesg | tail -50
+    sudo dmesg | tail -50
```

Comment:
We can get read darwin logs without sudo with builtin `log` utility. e.g. to get past five minutes kernel logs:
`log show --debug --predicate 'process == "kernel"' --start "$(date -v -5M '+%Y-%m-%d %H:%M:%S')"`